libs/execution_context/ebpf_link.c

```diff
@@ -137,6 +141,11 @@ ebpf_link_detach_program(ebpf_link_t* link)
 {
     ebpf_lock_state_t state;
     ebpf_program_t* program;
+
+    if (!link) {
+        return;
+    }
```

Comment:
My opinion is this would be better done by the caller and add `_In_` (not `_In_opt_`) SAL annotation to this function.